integreat_cms/cms/templates/events/event_list.html

```diff
@@ -73,9 +73,11 @@ <h1 class="heading">
                class="w-full mt-4 rounded border border-solid border-gray-200 shadow bg-white">
             <thead>
                 <tr class="border-b border-solid border-gray-200">
-                    <th class="py-3 pl-4 pr-2 min">
-                        <input form="bulk-action-form" type="checkbox" id="bulk-select-all" />
-                    </th>
+                    {% if perms.cms.change_event %}
+                        <th class="py-3 pl-4 pr-2 min">
+                            <input form="bulk-action-form" type="checkbox" id="bulk-select-all" />
+                        </th>
+                    {% endif %}
```

Comment:
I think the simplest fix for the padding is to simply include the `<th>` always and only hide its child:
```suggestion
                    <th class="py-3 pl-4 pr-2 min">
                        {% if perms.cms.change_event %}
                            <input form="bulk-action-form" type="checkbox" id="bulk-select-all" />
                        {% endif %}
                    </th>
```